webapp/javascript/components/TimelineChart/TimelineChartWrapper.tsx

```diff
@@ -148,6 +148,7 @@ class TimelineChartWrapper extends React.Component<
         //   a position and a nearby data item object as parameters.
         clickable: true,
       },
+      annotations: null,
```

Comment:
i think a default value is simply an empty array no?

Reply:
we have to separate cases 
- `annotations: []` (empty array) - we allow using annotation for this timeline, but at this moment annotations list is empty, user can add some
- `annotations: [Annotation, Annotation]` (non-empty array) - the same here, user can see and add annotations
- `annotations: null` - disable annotations functionality

Reply:
what's the difference between annotations functionality being "disabled" and simply no annotation existing?

Reply:
>"disabled"

user can't see and create annotations, whole annotations functionality is disabled

> simply no annotation existing

annotations list is empty, but user can add some ones